source/extensions/network/dns_resolver/cares/dns_impl.cc

```diff
@@ -243,7 +249,7 @@ void DnsResolverImpl::AddrInfoPendingResolution::onAresGetAddrInfoCallback(
       bool can_process_v6 =
           (!parent_.filter_unroutable_families_ || available_interfaces_.v6_available_);
 
-      int min_ttl = INT_MAX; // [RFC 2181](https://datatracker.ietf.org/doc/html/rfc2181)
+      int32_t min_ttl = INT_MAX; // [RFC 2181](https://datatracker.ietf.org/doc/html/rfc2181)
```

Comment:
It's a bit odd that the data type is `int32_t` but is initialized to `INT_MAX`. Perhaps we should use `std::numeric_limits` to get the types to agree?

Reply:
Addressed in https://github.com/envoyproxy/envoy/pull/38559/commits/9141e63116f6ed20494062938928db2662a0b5be